### PR TITLE
Fix sitemap

### PR DESCRIPTION
**Pull request by calumgunn**

### CODE REVIEW

**Comment by calumgunn** [comment]
@bensymonds @erino Cool, cool

**Comment by bensymonds** [comment]
Thinking about what's best for commit messages and stuff, I agree with https://github.com/scalaz/scalaz/wiki/Backporting-guidelines that the PR name is a good candidate for commit message. So the commit message could read:

```
Backport: fix sitemap
#152
```

or something.

I really don't know what's best at the moment.

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,8 @@
+* [#154](https://github.com/withassociates/slices/pull/154): Fix issue where
```
Comment (bensymonds) [inline CHANGELOG.md]: How about adding the `### Bug fixes` heading? Then the person doing the release doesn't have to figure that bit out (but they _will_ have to decide which part of version number to bump).

**Comment by bensymonds** [comment]
Made a comment re. changelog, otherwise :+1:

**Pull request merged**